src/components/Dropdown/DropdownMenu.react.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import * as React from "react";
+import cn from "classnames";
+
+import { Icon } from "../";
+
+type Props = {|
+  +children?: React.Node,
+  +className?: string,
+  +position?: string,
+  +arrow?: boolean,
+|};
+
+function DropdownMenu({
+  className,
+  position = "",
+  arrow,
+  ...rest
+}: Props): React.Node {
+  const classes = cn(
+    {
+      "dropdown-menu": true,
+      [`dropdown-menu-${position}`]: position,
+      [`dropdown-menu-arrow`]: arrow,
+    },
+    className
+  );
+  return <div className={classes} {...rest} />;
```

Comment:
Should we be rendering the children here?

Reply:
So in this case, children are a nullable type which means that they will either be `null | React.Node` I believe, which removes the case where they are undefined.